contracts/src/StakeTableV2.sol

```diff
@@ -94,32 +101,130 @@ contract StakeTableV2 is StakeTable, PausableUpgradeable, AccessControlUpgradeab
         return (2, 0, 0);
     }
 
+    /// @notice Pause the contract
+    /// @dev This function is only callable by the PAUSER_ROLE
     function pause() external onlyRole(PAUSER_ROLE) {
         _pause();
     }
 
+    /// @notice Unpause the contract
+    /// @dev This function is only callable by the PAUSER_ROLE
     function unpause() external onlyRole(PAUSER_ROLE) {
         _unpause();
     }
 
+    /// @notice Withdraw previously delegated funds after a validator has exited
+    /// @param validator The validator to withdraw from
+    /// @dev This function is overridden to deduct the amount from the validator's delegatedAmount
+    /// @dev and to add pausable functionality
+    /// @dev since the delegated Amount is no longer updated during validator exit
     function claimValidatorExit(address validator) public virtual override whenNotPaused {
-        super.claimValidatorExit(validator);
+        address delegator = msg.sender;
+        uint256 unlocksAt = validatorExits[validator];
+        if (unlocksAt == 0) {
+            revert ValidatorNotExited();
+        }
+
+        if (block.timestamp < unlocksAt) {
+            revert PrematureWithdrawal();
+        }
+
+        uint256 amount = delegations[validator][delegator];
+        if (amount == 0) {
+            revert NothingToWithdraw();
+        }
+
+        // Mark funds as spent
+        delegations[validator][delegator] = 0;
+        // deduct the amount from the validator's delegatedAmount
```

Comment:
I think this comment just explains in other words what the code is doing and can be removed.

We should however add a comment like `// new in V2`.

Please also add the changes to the summary of changes comment on top of the contract.